crates/iota/src/name_commands.rs

```diff
@@ -1086,36 +1403,198 @@ impl std::fmt::Display for NameCommandResult {
-                        .format("%Y-%m-%d %H:%M:%S.%f UTC")
-                        .to_string();
+fn format_registry_entry(f: &mut std::fmt::Formatter, entry: &RegistryEntry) -> std::fmt::Result {
+    let data = [
+        ("ID", entry.id.to_string()),
+        ("Domain", entry.domain.to_string()),
+    ];
+    let mut table_builder = Table::builder(data);
+    table_builder.set_header(["field", "value"]);
+
+    build_name_record_table(&mut table_builder, &entry.name_record);
+
+    let mut table = table_builder.build();
+    table.with(
+        tabled::settings::Style::rounded().horizontals([HorizontalLine::new(
+            1,
+            TableStyle::modern().get_horizontal(),
+        )]),
+    );
+    write!(f, "{table}")
+}
 
-                    table_builder.push_record([
-                        nft.id().to_string(),
-                        nft.domain_name().to_owned(),
-                        format!("{} ({expiration_datetime})", nft.expiration_timestamp_ms()),
-                    ]);
-                }
+fn format_reverse_registry_entry(
+    f: &mut std::fmt::Formatter,
+    entry: &ReverseRegistryEntry,
+) -> std::fmt::Result {
+    let data = [
+        ("ID", entry.id.to_string()),
+        ("Address", entry.address.to_string()),
+        ("Domain", entry.domain.to_string()),
+    ];
+    let mut table_builder = Table::builder(data);
+    table_builder.set_header(["field", "value"]);
+    let mut table = table_builder.build();
+
+    table.with(
+        tabled::settings::Style::rounded().horizontals([HorizontalLine::new(
+            1,
+            TableStyle::modern().get_horizontal(),
+        )]),
+    );
+    write!(f, "{table}")
+}
 
-                let mut table = table_builder.build();
-                table.with(
-                    tabled::settings::Style::rounded().horizontals([HorizontalLine::new(
-                        1,
-                        TableStyle::modern().get_horizontal(),
-                    )]),
-                );
-                write!(f, "{table}")
-            }
-        }
+fn format_name_record(f: &mut std::fmt::Formatter, record: &NameRecord) -> std::fmt::Result {
+    let mut table_builder = TableBuilder::default();
+
+    build_name_record_table(&mut table_builder, record);
+    table_builder.set_header(["field", "value"]);
+
+    let mut table = table_builder.build();
+    table.with(
+        tabled::settings::Style::rounded().horizontals([HorizontalLine::new(
+            1,
+            TableStyle::modern().get_horizontal(),
+        )]),
+    );
+    write!(f, "{table}")
+}
+
+fn build_name_record_table(table_builder: &mut TableBuilder, record: &NameRecord) {
+    table_builder.push_record(["NFT ID", record.nft_id.bytes.to_string().as_str()]);
+    table_builder.push_record([
+        "Target Address",
+        record
+            .target_address
+            .map(|address| address.to_string())
+            .unwrap_or_else(|| "<n/a>".to_owned())
```

Comment:
```suggestion
            .unwrap_or_else(|| "none".to_owned())
```
I don't think it's really Non Applicable 😬